src/ITensorMPS/ITensorMPS.jl

```diff
@@ -0,0 +1,34 @@
+module ITensorMPS
+
+using Adapt
+using HDF5
+using ..ITensors
+using IsApprox
+using KrylovKit
+using Printf
+using Random
+using TupleTools
```

Comment:
We could move these to a `usings.jl` file, to match the style of how we handle imports and exports.

Reply:
That's a good idea... let's do it